### ISSUE TITLE
Disable Column sorting

### ISSUE THREAD

**Issue by marcemira** [ChrisHonniball/ember-sortable-table#3]
Hey, awesome addon. Can you please add support for disabling columns? I think you can use a class for th or an array of columns positions, according with the plugin documentation.

PS. tried with that weird metadata on class method, but it didn't worked.

Cheers

**Comment by ChrisHonniball** [comment]
I just make the column that I don't want to sort a `<td>` rather than a `<th>` to get this functionality.

Yes, I can take a look at this when I get some time.

**Comment by marcemira** [comment]
@ChrisHonniball yeah, did the same hehe, but is not a good practice to use td on thead, that's why I asked :). Thanks you !

**Comment by ChrisHonniball** [comment]
I'll probably simply add an option `disableSortClass` to handle this and make it default to `disabled`. So that all you'll need to do is add that class to the `<th>` but it'll also give you the flexibility to use your own class name as well.

**Comment by marcemira** [comment]
@ChrisHonniball Cool thanks :grin: :+1:
